### ISSUE TITLE
Can not seem to have a function that returns an any value???!

### ISSUE THREAD

**Issue by rwilcox** [vlang/v#12949]
**V doctor:**
```
OS: linux, Ubuntu 20.04.3 LTS
Processor: 4 cpus, 64bit, little endian, Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
CC version: cc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0

getwd: /home/aias/Development/_Learning/v_isolate
vmodules: /home/aias/.vmodules
vroot: /home/aias/Applications/v
vexe: /home/aias/Applications/v/v
vexe mtime: 2021-12-23 19:08:30
is vroot writable: true
is vmodules writable: true
V full version: V 0.2.4 68452cf

Git version: git version 2.25.1
Git vroot status: weekly.2021.51-33-g68452cff
.git/config present: true
thirdparty/tcc status: thirdparty-linux-amd64 3654d6a8

```

**What did you do?**
`v -g -o vdbg cmd/v && vdbg v_isolate.v`
```v
module main


fn can_not_have_a_function_that_returns_an_any() any {
   return "1"
}

fn main() {
    can_not_have_a_function_that_returns_an_any()
}

```

**What did you expect to see?**

successful compilation

**What did you see instead?**
```
==================
/tmp/v_1000/v_isolate.7548883054729346987.tmp.c:1773: error: ';' expected (got "main__can_not_have_a_function_that_returns_an_any")
...
==================
(Use `v -cg` to print the entire error message)

builder error: 
==================
C error. This should never happen.

This is a compiler bug, please report it using `v bug file.v`.

https://github.com/vlang/v/issues/new/choose

You can also use #help on Discord: https://discord.gg/vlang

```

**Comment by JalonSolov** [comment]
@medvednikov Are we still thinking of removing `any` completely?

**Comment by felipensp** [comment]
Currently it prints:
```
bug.v:1:50: error: cannot use `any` type here, `any` will be implemented in V 0.4
    1 | fn can_not_have_a_function_that_returns_an_any() any {
      |                                                  ~~~
    2 |    return "1"
    3 | }
```
